### DISCUSSION TITLE
How to set up a template to append under a heading?

### DISCUSSION THREAD

**Discussion by ballastcanary** [coddingtonbear/obsidian-web#80] [Q&A] [answered]
I've tried this in the Local Rest API and it works with the demo.  But how do you set up your own Obsidian Web template that does this?  I want to be able to easily clip page title, URL and some highlighted text to a heading in the daily note called # Articles.
At present I can only see how to append to the end of the note.

**Comment by coddingtonbear** [comment, marked as answer]
You can use the `PATCH` method as documented here: https://coddingtonbear.github.io/obsidian-local-rest-api/#/Vault%20Files/patch_vault__filename_

**Reply by ballastcanary** [reply]
Thanks - tried 
PATCH  /periodic/daily/Articles
"Could not send content to Obsidian: (Error Code 40400) Not Found"

**Reply by coddingtonbear** [reply]
I believe you may want to look at the documentation at the link I provided once again; because I don't think the URL you're using -- `/periodic/daily/Articles` -- exists.
